app/views/layouts/listnav/_ems_cluster.html.haml

```diff
@@ -67,9 +67,9 @@
             :record_id    => @record.id,
             :display      => 'resource_pools',
             :title        => _("Show Resource Pools"))
-        - if role_allows?(:feature => "resource_pool_show_list")
```

Comment:
that was checking rbac on wrong feature as it's handling the drift history not resource pool list view (so yes, it was bug)